src/cow-react/modules/limitOrders/utils/getDecimals.ts

```diff
@@ -0,0 +1,11 @@
+import { DEFAULT_DECIMALS } from '@src/custom/constants'
+import { Currency } from '@uniswap/sdk-core'
+
+export function getDecimals(currency: Currency): number {
+  const mistakes = ['ALI']
```

Comment:
What does it mean?

Reply:
Good question, for some reason this token has defined 8 decimals but when that value is used the result is wrong and works with 18 decimals instead. 
![Screenshot from 2022-11-04 10-46-20](https://user-images.githubusercontent.com/34926005/199943137-5c6b10d8-36b4-4b57-bc8d-cc4085492595.png)